inst/extdata/config.yaml

```diff
@@ -1,58 +1,40 @@
 # This is a config file for the piccr package
 
-FILE_ID: "*.csv"	# file extension used for data files
-COLUMNSEP: ","		# column separator used in data files
-DECSEP: "."   		# decimal separator used in data files
```

Comment:
`COLUMNSEP` und `DECSEP` sind meiner Ansicht nach nicht mehr relevant, weil der Picarro Output einem einheitlichen Standard folgt. Gibt es andere Geräte, die nicht die englische Notation verwenden?

Reply:
Ok for now; we just have to keep in mind that different Picarro devices could use different column names in their output files (bremerhaven user story) which I will try to find out.